modules/nw-route-specific-annotations.adoc

```diff
@@ -41,7 +41,7 @@ Note: Using this annotation provides basic protection against denial-of-service
 |`haproxy.router.openshift.io/ip_allowlist`
 | Sets an allowlist for the route. The allowlist is a space-separated list of IP addresses and CIDR ranges for the approved source addresses. Requests from IP addresses that are not in the allowlist are dropped.
 
-The maximum number of IP addresses and CIDR ranges directly visible in the `haproxy.config` file is 61. [^1^] |
+The maximum number of IP addresses and CIDR ranges directly visible in the `haproxy.config` file is 61. [^2^] |
```

Comment:
This is a nit, but do we need the word "directly" here?

Reply:
i don't really know, i didn't edit this part of the file 😅 just changed the citation number. i'm inclined to leave it as is

Reply:
That's fair.